### PR TITLE
Validate name space not empty compiler

### LINKED ISSUE #146: precompile of flow without namespace returns wrong id
When calling slangCompiler.preCompile with a flow that doesn't have a namespace (the line is missing), and then calling getId() on the returned Executable, it returns "null.flowName".

### PR DESCRIPTION
**Pull request by meirwah**
to fix #146

### CODE REVIEW

File: score-lang-compiler/src/main/java/org/openscore/lang/compiler/scorecompiler/ScoreCompilerImpl.java [hunk 1]
```diff
@@ -50,6 +50,7 @@
 
     @Override
     public CompilationArtifact compile(Executable executable, Set<Executable> path) {
+        Validate.notEmpty(executable.getNamespace(),"Operation/Flow must have a namespace");
```
Comment (meirwah) [inline score-lang-compiler/src/main/java/org/openscore/lang/compiler/scorecompiler/ScoreCompilerImpl.java]: @oritstone / @orius123  is this apporach valid? or shoyld be a varifier thing only???

Comment (orius123) [inline score-lang-compiler/src/main/java/org/openscore/lang/compiler/scorecompiler/ScoreCompilerImpl.java]: Looks great to me.

Comment (orius123) [inline score-lang-compiler/src/main/java/org/openscore/lang/compiler/scorecompiler/ScoreCompilerImpl.java]: Just add the flow or operation name to the message

**Comment by meirwah** [comment]
@oritstone will you merge this after short review?

**Comment by meirwah** [comment]
@einatneu does this solves your issue ?

**Comment by einatneu** [comment]
@meirwah - I need it to be in the precompile also

**Comment by meirwah** [comment]
added @einatneu ... check now...

**Comment by einatneu** [comment]
@meirwah - looks good, thanks!

**Pull request merged**